OctopusDSC/DSCResources/cTentacleAgent/cTentacleAgent.psm1

```diff
@@ -501,8 +565,22 @@ function Test-TargetResource {
     # Check Ensure value
     if ($Ensure -eq "Present" -and ![string]::IsNullOrEmpty($OctopusServerUrl))
     {
+        if (![string]::IsNullOrEmpty($Space))
+        {
+            # Get reference to the space
+            $spaceRef = Get-Space -Space $Space -ServerUrl $OctopusServerUrl -APIKey $ApiKey
+
+            # Check to see if something was returned
+            if ($null -eq $spaceRef)
+            {
+                # Throw error, space wasn't found
+                throw "Unable to find a space by the name of $Space"
```

Comment:
I always like to put single quotes around values like this - helps figure out what there's an issue where `$space` is set to null or a blank string.
```suggestion
                throw "Unable to find a space by the name of '$Space'"
```

Also, might be worth considering putting some help text in to say how to solve the problem?